addon.json

```diff
@@ -8,24 +8,26 @@
         },
         "exports": {
             "app": [
+                "./src/scripts/application",
                 "./src/scripts/events",
                 "./src/scripts/gdn",
                 "./src/scripts/permissions",
                 "./src/scripts/utility",
                 "./src/scripts/dom-utility",
                 "axios",
                 "react",
```

Comment:
We should probably add a couple more pieces here as well. `react-dom`, `react-redux`, `react-router-dom` and `redux`.